src/main/java/org/m2n/webapplications2/database/helpers/DbInit.java

```diff
@@ -56,10 +56,10 @@ public static void insertTestData() throws DatabaseException {
                 "VALUES ('Default');");
             statement.addBatch("INSERT INTO category (name, tagline, description)\n" +
                 "VALUES\n" +
-                "('Python', 'Dunno', 'Python is awesome'),\n" +
-                "('Food', 'Dunno', 'Eat! Or you''ll starve!'),\n" +
-                "('Health', 'Dunno', 'See category Food to stay healthy'),\n" +
-                "('Hardware', 'Dunno', 'Hardware Hacking Topics');\n");
+                "('Python', 'Programming Language', 'Python is awesome'),\n" +
+                "('Food', 'Maultaschen', 'Eat! Or you''ll starve!'),\n" +
+                "('Health', 'Sports', 'See category Food to stay healthy'),\n" +
+                "('Hardware', 'Security', 'Hardware Hacking Topics');\n");
```

Comment:
Please also change the corresponding `.sql` file (should be `/src/main/sql/insert-test.sql` or something like that)

Reply:
Done. 
429d830

Reply:
Thanks 👍